### PR TITLE
Clafirication on prerequisites

### PR DESCRIPTION
**Pull request by chiedojohn**
Here you go!

### CODE REVIEW

**Comment by chiedojohn** [comment]
Any updates on this?

File: readme.md [hunk 1]
```diff
@@ -39,6 +39,13 @@ chrome_options.add_argument("--proxy-server={0}".format(proxy.proxy))
 browser = webdriver.Chrome(chrome_options = chrome_options)
 ```
 
+Before Running Tests
```
Comment (AutomatedTester) [inline readme.md]: This part shouldnt be required, in https://github.com/AutomatedTester/browsermob-proxy-py/blob/master/browsermobproxy/server.py#L53  the code should start up the process.

Comment (chiedojohn) [inline readme.md]: Hmmm... I'm not sure what was going on there for me then. That was the only way I was able to get it to work. Maybe something wrong with my setup?

Comment (AutomatedTester) [inline readme.md]: Possibly. Are you passing in the full path to the binary?

**Comment by chiedojohn** [comment]
I tried both the full path and relative path.

**Comment by AutomatedTester** [comment]
can you raise a bug, with a reduced test case, so that I can try recreate
the issue. Not sure what is special in your setup to warrant this.

**Comment by chiedojohn** [comment]
Hey man!
Here is the error

```
File "run.py", line 61, in <module>
    server.start()
  File "/usr/local/Cellar/python/2.7.6/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/browsermobproxy/server.py", line 63, in start
    raise Exception("Can't connect to Browsermob-Proxy")
```

I am running the following from https://github.com/chiedojohn/python-selenium-starter/tree/b30fb35b9b3dd8fa2dbe31cbf26286791e152a98
```
python run.py --proxy
```

No big deal if you can't figure it out though as the workaround I have been using is only a minor inconvenience.

**Comment by AutomatedTester** [comment]
I can't get your version of bmp to start up but I made a small change

```patch
diff --git a/run.py b/run.py
index 051b54b..bf13cf4 100644
--- a/run.py
+++ b/run.py
@@ -57,7 +57,9 @@ else:
 # Set up the browsermob proxy if the argument is passed
 if(args.proxy):
     from browsermobproxy import Server
-    server = Server("browsermob-proxy/bin/browsermob-proxy")
+    path_to_bmp = os.path.abspath(os.path.join(os.path.dirname(__file__),
"browsermob-proxy/bin"))
+    server = Server("%s/browsermob-proxy" % path_to_bmp)
     server.start()
     proxy = server.create_proxy()
```

The error I was getting is

```
dburns in /private/tmp/python-selenium-starter/browsermob-proxy/bin on
master ● λ ./browsermob-proxy
Exception in thread "main" java.lang.NoClassDefFoundError:
net/lightbody/bmp/proxy/Main
Caused by: java.lang.ClassNotFoundException: net.lightbody.bmp.proxy.Main
    at java.net.URLClassLoader$1.run(URLClassLoader.java:202)
    at java.security.AccessController.doPrivileged(Native Method)
    at java.net.URLClassLoader.findClass(URLClassLoader.java:190)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:306)
    at sun.misc.Launcher$AppClassLoader.loadClass(Launcher.java:301)
    at java.lang.ClassLoader.loadClass(ClassLoader.java:247)
```

David Burns
Email: david.burns@theautomatedtester.co.uk
URL: http://www.theautomatedtester.co.uk/

**Comment by chiedojohn** [comment]
Thanks David,
I went ahead and just re-downloaded browsermob proxy and that did the trick. It may have needed the jar files present.

**Comment by AutomatedTester** [comment]
awesome! glad it was that simple to solve ;)

David Burns
Email: david.burns@theautomatedtester.co.uk
URL: http://www.theautomatedtester.co.uk/

**Comment by chiedojohn** [comment]
Today I bow my head in shame. LOL

**Pull request closed without merging**